openmc/mgxs/library.py

```diff
@@ -1031,10 +1029,10 @@ def check_library_for_openmc_mgxs(self):
         # Total or transport can be present, but if using
         # self.correction=="P0", then we should use transport.
         if (((self.correction is "P0") and
-             ('transport' not in self.mgxs_types))):
+             ('nu-transport' not in self.mgxs_types))):
             error_flag = True
-            msg = 'Transport MGXS type is required since a "P0" correction ' \
-                  'is applied, but a Transport MGXS is not provided.'
+            msg = 'NuTransport MGXS type is required since a "P0" correction' \
```

Comment:
Perhaps we re-word to more closely match our accepted MGXS type strings: "A "nu-transport" MGXS type"...